.goreleaser.yml

```diff
@@ -93,4 +93,33 @@ release:
     owner: redpanda-data
     name: connect
   prerelease: auto
-  disable: false
+  disable: true
```

Comment:
```suggestion
  disable: false
```

I'm assuming this is a left over from testing.

Reply:
yes great catch - serious one.